tests/func-tests/monitoring_test.go

```diff
@@ -144,11 +144,15 @@ func getAlertByName(alerts promApiv1.AlertsResult, alertName string) *promApiv1.
 
 func verifyOperatorHealthMetricValue(promClient promApiv1.API, initialOperatorHealthMetricValue, alertImpact float64) {
 	Eventually(func() bool {
-		systemHealthMetricValue := getMetricValue(promClient, "kubevirt_hco_system_health_status")
-		operatorHealthMetricValue := getMetricValue(promClient, "kubevirt_hyperconverged_operator_health_status")
+		if initialOperatorHealthMetricValue <= alertImpact {
+			systemHealthMetricValue := getMetricValue(promClient, "kubevirt_hco_system_health_status")
+			operatorHealthMetricValue := getMetricValue(promClient, "kubevirt_hyperconverged_operator_health_status")
+			expectedOperatorHealthMetricValue := math.Max(alertImpact, systemHealthMetricValue)
 
-		expectedOperatorHealthMetricValue := math.Max(alertImpact, math.Max(systemHealthMetricValue, initialOperatorHealthMetricValue))
-		return operatorHealthMetricValue == expectedOperatorHealthMetricValue
+			return operatorHealthMetricValue == expectedOperatorHealthMetricValue
```

Comment:
Please use another aay to compare these values, in a way that will produce meaningful error message  in case of failure.

You can follow the implementation in #2378